code/modules/research/designs.dm

```diff
@@ -50,16 +50,18 @@ other types of metals and chemistry for reagents).
 	CRASH("DESIGN DATUMS SHOULD NOT EVER BE DESTROYED AS THEY ARE ONLY MEANT TO BE IN A GLOBAL LIST AND REFERENCED FOR US.")
 	return ..()
 
-/datum/design/proc/icon_html(client/user)
-	if (!icon_cache)
+/datum/asset/simple/research_designs/register()

```

Comment:
Move this together with the other asset caches, having this random path in the file that has nothing to do with it is really silly and annoying.

Reply:
Addressed.